plugin.xml

```diff
@@ -59,7 +59,7 @@
         <config-file target="AndroidManifest.xml" parent="application">
           <provider
               android:name="org.apache.cordova.camera.FileProvider"
-              android:authorities="${applicationId}.provider"
+              android:authorities="${applicationId}.cordova.plugin.camera.provider"
```

Comment:
Is this considered a breaking change for existing applications?

Reply:
Maybe....

https://developer.android.com/guide/topics/manifest/provider-element

Looks like this provider authority name is used in `content://` urls, but I believe these are just temporarily paths anyway...? Wouldn't you read the path and save it to a more persistent location, generally speaking?

Reply:
I don't think it's breaking, but we are doing a major release anyway, so it won't be a problem anyway. I fixed some conflicts it had, going to merge.

We should probably add a provider in cordova-android for plugins to use instead of each plugin having it's own, but providers need an Android support/Android X dependency, so not sure if we want that in cordova-android.
If not, just a simple official plugin that only have the provider and others can use as a dependency.